recipes/gtest/all/conanfile.py

```diff
@@ -53,18 +54,31 @@ def _minimum_compilers_version(self):
     def _is_clang_cl(self):
         return self.settings.os == "Windows" and self.settings.compiler == "clang"
 
+    def export_sources(self):
+        if Version(self.version) < "1.12.0":
+            for patch in self.conan_data.get("patches", {}).get(self.version, []):
+                copy(self, patch["patch_file"], self.recipe_folder, self.export_sources_folder)
```

Comment:
```suggestion
        export_conandata_patches(self)
```

https://docs.conan.io/en/latest/reference/conanfile/tools/files/patches.html#conan-tools-files-export-conandata-patches